### PR TITLE
 Register Project Wednesday (Playground AYOD) RAG Model in assets

### PR DESCRIPTION
**Pull request by anushree1808**
Register Project Wednesday (Playground AYOD) RAG Model assets in the repo.

### CODE REVIEW

File: assets/promptflow/models/playground-ayod-rag/model.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+path:
+  container_name: models
+  container_path: samples/playground_ayod_rag/v1/models
+  storage_name: wedpromptflowragsamples
```
Comment (vizhur) [inline assets/promptflow/models/playground-ayod-rag/model.yaml:4]: is this a public storage?

Comment (anushree1808) [reply assets/promptflow/models/playground-ayod-rag/model.yaml:4]: The storage has been assigned anonymous blob access as well as anonymous level permission is set at container level. Followed the instruction from [here](https://msdata.visualstudio.com/Vienna/_git/PromptFlow?path=/docs/sharing-your-flows-in-prompt-flow-gallery.md&version=GBgega/flowsharingexamples&_a=preview&anchor=4.1.1-uploading-your-flow-to-azure-blob-container).

Comment (vizhur) [reply assets/promptflow/models/playground-ayod-rag/model.yaml:4]: It might be better to have it private?

**Review by vizhur — CHANGES_REQUESTED** [review]
lets use private storages

File: assets/promptflow/models/playground-ayod-rag/model.yaml (hunk 1 above)
Comment (anushree1808) [reply assets/promptflow/models/playground-ayod-rag/model.yaml:4]: Hi @vizhur, I have updated the storage account access now:
- removed anonymous blob permission
- add permission for test-agent (storage blob reader)

image

**Review by vizhur — APPROVED** [review]

**Comment by anushree1808** [comment]
> lets use private storages

**Pull request merged**